dotnet/samples/GettingStartedWithProcesses/Models/AccountUserInteractionDetails.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel;
+
+namespace Models;
+
+public class AccountUserInteractionDetails
```

Comment:
Please make the classes private and sealed and add XML comments at the class level.

Reply:
added missing XML comments for the model and event classes.
Since these are in a separate file leaving them as public, also setting those models and steps public would allow users to use the step samples outside the GettingStartedWithProcesses, similar to Concepts/Plugins

Reply:
Ok, makes sense. Thanks.

Reply:
Note: I believe `private` is only valid for an _inner_ class definition.  I think `internal` would be the available option...should it be chosen.